packages/css-library/src/tokens/visibility.scss

```diff
@@ -0,0 +1,39 @@
+$tokens-visibility: (
+  'hidden': hidden,
+  'visible': visible,
+);
+
+.vads-visibility-screen-reader {
+  border: 0;
+  clip: rect(0, 0, 0, 0);
+  clip-path: inset(50%);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+  word-wrap: normal;
+}
+
+$u-breakpoints: (
+  'mobile-lg': 30em,
+  'tablet': 40em,
+  'desktop': 64em,
+);
+@each $media-key, $media-value in $u-breakpoints {
+  @media all and (min-width: #{$media-value}) {
+    .#{$media-key}\:vads-visibility-screen-reader {
+      border: 0;
+      clip: rect(0, 0, 0, 0);
+      clip-path: inset(50%);
+      height: 1px;
+      margin: -1px;
+      overflow: hidden;
+      padding: 0;
+      position: absolute;
+      width: 1px;
+      word-wrap: normal;
+    }
+  }
+}
```

Comment:
There's a lot of [extra things in here coming from some blog post](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/a20b0958ebf8ac9bb6bd1630638678297eb83cfe/packages/formation/sass/utilities/_visibility.scss#L9-L23). Do we need all of them, or could we [`@extend usa-sr-only`](https://github.com/uswds/uswds/blob/846e45b50662851a6d37f20726d9d20dd415a3fa/packages/uswds-helpers/src/styles/_usa-sr-only.scss#L3-L5). The internal mixin looks [much simpler](https://github.com/uswds/uswds/blob/846e45b50662851a6d37f20726d9d20dd415a3fa/packages/uswds-core/src/styles/mixins/general/screen-reader.scss#L2-L6).

Also, we might not need this to be responsive. If something should only be visible to screen readers in a desktop view, I'm guessing the same would be true for tablet, mobile, etc.

Reply:
Good call on this not needing to be responsive for the screen reader utility, and thank you for those helpful links.